### DISCUSSION TITLE
Problem load radio content in to Edge Companion

### DISCUSSION THREAD

**Discussion by jacksoderberg** [EdgeTX/edgetx#2572] [Q&A] [answered]
TX16MKII 
Edge 2.7
Companion 2.7

When trying to load radio content message in Companion I get the error message
"cant load models, end of map not found?

I only have Yml files in model and radiofolder

Please help out if possible 
see attaches picture
<img width="1280" alt="Skärmavbild 2022-10-14 kl  12 42 40" src="https://user-images.githubusercontent.com/115811044/195970652-890acd5d-8bc7-44bd-a0a7-7fb567b4eac1.png">

**Comment by raphaelcoeffic** [comment, marked as answer]
These models files are faulty because they use an empty file name for some special functions. If you open the file, you will see some strings looking like `,1x`. I believe this has been fixed in `2.7.1`. The model file can be repaired either manually by editing the file and replacing `,1x` with `",1x"`, or just opening the model on radio with a fixed firmware (>=`2.7.1`).

**Reply by pfeerick** [reply]
Specially, check out the "Eachine P-51D" model... this has been brought up a few times now - most recently in https://github.com/EdgeTX/edgetx/discussions/2350 which also links to the earlier discussion/fixing of.

**Reply by jacksoderberg** [reply]
> Specially, check out the "Eachine P-51D" model... this has been brought up a few times now - most recently in #2350 which also links to the earlier discussion/fixing of.

Yes, that did it, I just shortened the name of the Mustang P51 to just P 51 and now companion loads models and radio data flawless.
Big thanks everybody
/Jack
